frontend/pages/hosts/ManageHostsPage/components/FilterPill/FilterPill.tsx

```diff
@@ -55,7 +58,7 @@ const FilterPill = ({
             place="bottom"
             effect="solid"
             backgroundColor="#3e4771"
-            id="filter-pill-tooltip"
+            id={`filter-pill-tooltip-${label}`}
```

Comment:
This was a gotcha as the tooltip was rendering on both pills with label id not unique